src/Utilities/Compiler/WellKnownTypeNames.cs

```diff
@@ -108,6 +108,7 @@ internal static class WellKnownTypeNames
         public const string SystemDataDataSet = "System.Data.DataSet";
         public const string SystemDataDataTable = "System.Data.DataTable";
         public const string SystemDataDataViewManager = "System.Data.DataViewManager";
+        public const string SystemDataEntityDbSetGenerics = "System.Data.Entity.DbSet`1";
```

Comment:
Please rename into `SystemDataEntityDbSet1`

Reply:
Please rename this one.